src/server.ts

```diff
@@ -15,7 +16,10 @@ const start = async () => {
     secret: process.env.PAYLOAD_SECRET,
     express: app,
     onInit: async () => {
-      payload.logger.info(`Payload Admin URL: ${payload.getAdminURL()}`)
+      payload.logger.info(`Payload Admin URL: http://localhost:3000${payload.getAdminURL()}`)
+      if (process.env.PAYLOAD_SEED === 'true') {
```

Comment:
maybe add this env variable to `pnpm run dev`?

Reply:
I think we don't need this condition anymore, because it now seeds always if the database is empty.